freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/user/model/SetPasswordResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.freeipa.api.v1.freeipa.user.model;
+
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModel;
+
+@ApiModel("SetPasswordV1Response")
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SetPasswordResponse {
+    private final String value;
```

Comment:
this "value" field is too ambiguous

Reply:
I added code to expose for which environments the password was successfully set